CMakeLists.txt

```diff
@@ -11,8 +11,8 @@ if (NOT CMAKE_BUILD_TARGET)
   set(CMAKE_BUILD_TARGET "Both" CACHE STRING "" FORCE)
 endif()
 
-option(BUILD_EXAMPLE "Build example program" OFF)
-option(BUILD_TEST "Build test program" OFF)
+option(BUILD_EXAMPLE "Build example program" ON)
+option(BUILD_TEST "Build test program" ON)
```

Comment:
You need to revert these changes; this is the cause of the Travis build failure.